app/src/components/dataset/DatasetExampleSelect.tsx

```diff
@@ -0,0 +1,95 @@
+import { Suspense, useMemo } from "react";
+import { useLazyLoadQuery } from "react-relay";
+import { graphql } from "relay-runtime";
+import { css } from "@emotion/react";
+
+import { Button } from "@phoenix/components/button";
+import { DatasetExampleSelect_ExampleSelectPopoverContentQuery } from "@phoenix/components/dataset/__generated__/DatasetExampleSelect_ExampleSelectPopoverContentQuery.graphql";
+import { SelectChevronUpDownIcon } from "@phoenix/components/icon";
+import { ListBox } from "@phoenix/components/listbox";
+import { Loading } from "@phoenix/components/loading";
+import { Popover } from "@phoenix/components/overlay";
+import { Select, SelectItem, SelectValue } from "@phoenix/components/select";
+import { Truncate } from "@phoenix/components/utility/Truncate";
+import { IdTruncate } from "@phoenix/pages/playground/PromptMenu";
+
+export type DatasetExampleSelectProps = {
+  datasetId: string | null;
+  selectedExampleId: string | null;
+  onSelectExampleId: (exampleId: string | null) => void;
+};
+
+export const DatasetExampleSelect = (props: DatasetExampleSelectProps) => {
+  const { selectedExampleId, onSelectExampleId } = props;
+  return (
+    <Select
+      selectionMode="single"
+      value={selectedExampleId}
+      onChange={(value) => onSelectExampleId(value as string | null)}
+      css={css`
+        min-width: 0;
+      `}
+      aria-label="Select an example"
+      placeholder="Select an example"
+    >
+      <Button
+        css={css`
+          min-width: 0 !important;
```

Comment:
I think compositionally shouldn't there be a way that this doesn't require the !important